apps/authentication/mfa/face.py

```diff
@@ -32,7 +33,10 @@ def is_active(self):
 
     @staticmethod
     def global_enabled():
-        return settings.XPACK_LICENSE_IS_VALID and settings.FACE_RECOGNITION_ENABLED
+        return settings.XPACK_LICENSE_IS_VALID \
+            and LicenseEditionChoices.ULTIMATE == \
+            LicenseEditionChoices.from_key(settings.XPACK_LICENSE_EDITION) \
+            and settings.FACE_RECOGNITION_ENABLED
 
     def get_enable_url(self) -> str:
         return reverse('authentication:user-face-enable')
```

Comment:
The main difference between this code block is that a new constant `LicenseEditionChoices` has been added to handle license edition choices other than the standard ultimate edition. The method `global_enabled()` now checks if XPack License and Face Recognition are both enabled.

Potential issue/indication of a bug:

1. A missing colon after the first instance of "settings." This makes Python syntax incorrect (no trailing comma).

Optimization suggestion / Improvement idea:

This could be improved further through unit tests verifying the functionality with input data points (e.g., valid/not-valid licenses), which would ensure the solution's reliability under different conditions.